runtime/vm/module.xml

```diff
@@ -186,9 +186,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 				<include-if condition="spec.flags.J9VM_OPT_ZIP_SUPPORT"/>
 			</library>
 			<library name="ffi"/>
-			<library name="j9jitbuilder">
-				<include-if condition="spec.flags.J9VM_OPT_PANAMA"/>
-			</library>
+			<library name="j9jitbuilder"/>
```

Comment:
The include condition here must match the one in `runtime/j9jitbuilder/module.xml`.

Reply:
Done